src/Generators/OrmTagGenerator.php

```diff
@@ -137,6 +143,22 @@ protected function generateHasManyTags()
         $this->generateTagsForDataLists($this->getClassConfig('has_many'), DataList::class);
     }
 
+    /**
+     * Generate the $many_many method values.
+     */
+    protected function generateManyManyTags()
+    {
+        $this->generateTagsForDataLists($this->getClassConfig('many_many'), ManyManyList::class);
```

Comment:
Does this method need to be updated, or a new one added, to support ManyManyThroughList?

Reply:
That's from the rebased version, right? The generateTagsForDataLists method takes care of the optional `through`